storage_server/report_server.py

```diff
@@ -526,11 +526,16 @@ def addSuppressBug(self, run_id, bugs_to_suppress):
         try:
             suppressList = []
             for bug_to_suppress in bugs_to_suppress:
-                suppress_bug = SuppressBug(run_id,
-                                           bug_to_suppress.bug_hash,
-                                           bug_to_suppress.file_name,
-                                           bug_to_suppress.comment)
-                suppressList.append(suppress_bug)
+                res = self.session.query(SuppressBug) \
+                    .filter(SuppressBug.hash == bug_to_suppress.bug_hash) \
```

Comment:
So copy-paste codes may still cause hash collision if they are in the same file, right? Or is line number part of the hash? Then why file name isn't? Anyway I should once check how a hash is generated :wink: